### DISCUSSION TITLE
[bug report] unscrollable modal on product tour page

### DISCUSSION THREAD

**Discussion by shrekuu** [vercel/community#4675] [Bug Report]
### Page to Investigate

https://vercel.com/product-tour

### Steps to Reproduce

<img width="1624" alt="image" src="https://github.com/vercel/community/assets/3136026/7cdedb0a-01ff-4c94-a3c6-4c9a2687f150">

The Contact Sales modal is not scrollable on [production tour](https://vercel.com/product-tour). Please check.

**Comment by amyegan** [comment]
Thanks for reporting this! I let the team know so we can get it fixed

**Comment by amyegan** [comment]
Fixed! Thanks again for reporting this 🙂
